docker-compose.yml

```diff
@@ -1,12 +1,12 @@
-version: '2'
+version: '2.1'
 
 services:
   sylius:
     container_name: ${NAME}-app
     build:
         context: .
         args:
-            - AS_UID=${AS_UID}
+          - AS_UID=${AS_UID:-33}
```

Comment:
@adepretis Do you think this is worth it to go to 2.1 file format? Shouldn't the default value in the Docker-File be enough?

If we want to do this, I would recommend to also enable default values for the `NAME` variable

Reply:
```NAME``` is defined in the Makefile - we could of course define ```AS_UID``` with default 33 in the Makefile too (which then doesn't require the 2.1 format).

Regarding your question (Dockerfile sufficient): nope ... if ```AS_UID``` is undefined (which it currently is for e.g. Mac OS environments), ```AS_UID``` gets passed as an ```ARG``` with no value, overloading the default in the ``Dockerfile``. Therefore the ``RUN`` command doesn't work because it ends up as ```RUN usermod -u www-data```.

For further pipeline integration we will have to modify something though (not yet), otherwise a CI-system running on Linux (most likely) will get its UID baked into the image ... and that is something I'd rather not like to have ;-)